docs/plugins/ingest-geoip.asciidoc

```diff
@@ -203,3 +203,18 @@ Which returns:
 }
 --------------------------------------------------
 // TESTRESPONSE
+
+[[ingest-geoip-settings]]
+===== Settings
+
+The geoip processor supports the following settings:
+
+`ingest.geoip.cache_enabled`::
+
+    Whether to enable caching of results. Defaults to `false`.
```

Comment:
Yeah. I looked at LS doc: https://www.elastic.co/guide/en/logstash/current/plugins-filters-geoip.html and it's a single setting, 1000 by default. So activated by default.